### ISSUE TITLE
Unnecessary packages & Build time perf.

### ISSUE THREAD

**Issue by sambacha** [freight-trust/webapp#182]
priority removal:
npm ERR! missing: core-js@1.2.7, required by fbjs@0.8.17
npm ERR! missing: core-js@2.6.11, required by recharts@1.8.5

remove:
- [ ] etlify-cms-core
- [ ] netlify-cms-backend-github
- [ ] netlify-cms-app
- [ ] eslint-config-carbon-base

build/ci:
- [ ] circle-ci build time
- [ ] netlify cache 

> resolve major dep. issues
```bash
$ npm ls core-js
@freight-trust/webapp@1.9.4 /Users/sbacha/gitlab-gatsby/master/monoapp
├─┬ UNMET DEPENDENCY gatsby@2.24.76
│ ├── UNMET DEPENDENCY core-js@3.6.5
│ └─┬ UNMET DEPENDENCY gatsby-cli@2.12.107
│   └─┬ UNMET DEPENDENCY yurnalist@1.1.2
│     └─┬ UNMET DEPENDENCY babel-runtime@6.26.0
│       └── UNMET DEPENDENCY core-js@2.6.11
├─┬ UNMET DEPENDENCY gatsby-plugin-mdx@1.2.46
│ └── UNMET DEPENDENCY core-js@3.6.5
├─┬ UNMET DEPENDENCY netlify-cms-app@2.12.27
│ └─┬ UNMET DEPENDENCY netlify-cms-widget-date@2.5.5
│   └─┬ UNMET DEPENDENCY react-datetime@2.16.3
│     └─┬ UNMET DEPENDENCY create-react-class@15.6.3
│       └─┬ UNMET DEPENDENCY fbjs@0.8.17
│         └── UNMET DEPENDENCY core-js@1.2.7
└─┬ UNMET DEPENDENCY recharts@1.8.5
  └── UNMET DEPENDENCY core-js@2.6.11

npm ERR! missing: gatsby@2.24.76, required by @freight-trust/webapp@1.9.4
npm ERR! missing: gatsby-plugin-mdx@1.2.46, required by @freight-trust/webapp@1.9.4
npm ERR! missing: netlify-cms-app@2.12.27, required by @freight-trust/webapp@1.9.4
npm ERR! missing: recharts@1.8.5, required by @freight-trust/webapp@1.9.4
npm ERR! missing: core-js@3.6.5, required by gatsby@2.24.76
npm ERR! missing: gatsby-cli@2.12.107, required by gatsby@2.24.76
npm ERR! missing: yurnalist@1.1.2, required by gatsby-cli@2.12.107
npm ERR! missing: babel-runtime@6.26.0, required by yurnalist@1.1.2
npm ERR! missing: core-js@2.6.11, required by babel-runtime@6.26.0
npm ERR! missing: core-js@3.6.5, required by gatsby-plugin-mdx@1.2.46
npm ERR! missing: netlify-cms-widget-date@2.5.5, required by netlify-cms-app@2.12.27
npm ERR! missing: react-datetime@2.16.3, required by netlify-cms-widget-date@2.5.5
npm ERR! missing: create-react-class@15.6.3, required by react-datetime@2.16.3
npm ERR! missing: fbjs@0.8.17, required by create-react-class@15.6.3
npm ERR! missing: core-js@1.2.7, required by fbjs@0.8.17
npm ERR! missing: core-js@2.6.11, required by recharts@1.8.5
```

**Comment by sambacha** [comment]
TLDR: remove netlify cms 
further analysis:

```bash
$ npm ls fbjs
@freight-trust/webapp@1.9.4 /Users/sbacha/gitlab-gatsby/master/monoapp
└─┬ UNMET DEPENDENCY netlify-cms-app@2.12.27
  └─┬ UNMET DEPENDENCY netlify-cms-widget-date@2.5.5
    └─┬ UNMET DEPENDENCY react-datetime@2.16.3
      └─┬ UNMET DEPENDENCY create-react-class@15.6.3
        └── UNMET DEPENDENCY fbjs@0.8.17

npm ERR! missing: netlify-cms-app@2.12.27, required by @freight-trust/webapp@1.9.4
npm ERR! missing: netlify-cms-widget-date@2.5.5, required by netlify-cms-app@2.12.27
npm ERR! missing: react-datetime@2.16.3, required by netlify-cms-widget-date@2.5.5
npm ERR! missing: create-react-class@15.6.3, required by react-datetime@2.16.3
npm ERR! missing: fbjs@0.8.17, required by create-react-class@15.6.3
```
```bash
$ npm ls create-react-class
@freight-trust/webapp@1.9.4 /Users/sbacha/gitlab-gatsby/master/monoapp
├─┬ UNMET DEPENDENCY netlify-cms-app@2.12.27
│ └─┬ UNMET DEPENDENCY netlify-cms-widget-date@2.5.5
│   └─┬ UNMET DEPENDENCY react-datetime@2.16.3
│     └── UNMET DEPENDENCY create-react-class@15.6.3
└─┬ UNMET DEPENDENCY netlify-cms-core@2.32.0
  └─┬ UNMET DEPENDENCY redux-notifications@4.0.1
    └─┬ UNMET DEPENDENCY react-redux@4.4.10
      └── UNMET DEPENDENCY create-react-class@15.6.3

npm ERR! missing: netlify-cms-app@2.12.27, required by @freight-trust/webapp@1.9.4
npm ERR! missing: netlify-cms-core@2.32.0, required by @freight-trust/webapp@1.9.4
npm ERR! missing: netlify-cms-widget-date@2.5.5, required by netlify-cms-app@2.12.27
npm ERR! missing: react-datetime@2.16.3, required by netlify-cms-widget-date@2.5.5
npm ERR! missing: create-react-class@15.6.3, required by react-datetime@2.16.3
npm ERR! missing: redux-notifications@4.0.1, required by netlify-cms-core@2.32.0
npm ERR! missing: react-redux@4.4.10, required by redux-notifications@4.0.1
npm ERR! missing: create-react-class@15.6.3, required by react-redux@4.4.10
```

**Comment by sambacha** [comment]
#### originally introduced in:

```diff
- Resolve "Netlify CMS + GitLab Backend" [`#39`](https://github.com/freight-trust/webapp/pull/39)
-- Resolve "Netlify CMS + GitLab Backend" [`#38`](https://github.com/freight-trust/webapp/pull/38)
-- Merge branch '14-netlify-cms-gitlab-backend' into 'master' [`#14`](https://github.com/freight-trust/webapp/issues/14)
```

**Comment by sambacha** [comment]
closed by https://github.com/freight-trust/webapp/pull/179
